docs/src/proposals/unified-scheduler.md

```diff
@@ -0,0 +1,96 @@
+# Unified Scheduler
+
+## Architechtural problem
+
+- there's two execution scheduling algorithms, and both are subotimmal for current tx load. 
+independent algorithms are exposing the so-called replayability risk
+
+also, increasing banking threads doesn't linearly scale to the number of cpu cores.
+ 
+## Current Transaction load pattern
+
+## Redefined scheduler's design goal
+
+saturate N cores with full of tx exec cycles.
+
+## Proposed Solution
+
+## why unify?: because simply using the same algo makes most sense.
+
+## Currnet implementation problem
+
+- why batching isn't good?
```

Comment:
I can definitely see that this may be the case for replay, but is it for banking as well? Still learning all the different parts of banking, but I thought one of the main benefits of batching for banking is that we record to PoH less often which is more efficient.

@carllin, any thoughts here?

Reply:
> I thought one of the main benefits of batching for banking is that we record to PoH less often which is more efficient.

that concern is correct, however...:

- poh recorder's lock contention can be reduced significantly (even can be eliminated maybe) once there is only a single unified thread recording to it. (ie. the scheduler thread)
- also, scheduler can just buffer txes to record to poh until lock conflicting tx is encountered.

that said, I thought main benefits of batching would be AccountsDb locking.